frontend/src/components/car-list-item/styles.module.scss

```diff
@@ -108,3 +108,37 @@
     margin: 0;
   }
 }
+
+.button {
+  height: 1.2rem;
+
+  &:hover {
+    cursor: pointer;
+  }
+}
+
+.iconButton {
+  padding: 0;
+  background: transparent;
+  border: none;
+
+  path {
+    transition: all 300ms;
+  }
+
+  &.isLiked,
+  &:hover {
+    path {
+      transition: all 300ms;
+      fill: var(--blue100);
+    }
+  }
+}
```

Comment:
Don't we already have the styles for the like button?

Reply:
I added common component llike button